torch/csrc/jit/register_prim_ops.cpp

```diff
@@ -1895,14 +1913,108 @@ RegisterOperators reg2({
           return 0;
         }),
 
+    Operator(
+        "aten::ceil(float a) -> int",
```

Comment:
This should return a `float` (and could you change `aten::floor` above to also return a `float`)? Casting a `float` to an `int` may truncate the value if its bigger than `std::numeric_limits<int64_t>::max()`. It's safer to make the user explicitly agree to truncate. 

```python
@torch.jit.script
def double_floor(x):
    # type: (float) -> float
    return x - x % 1

@torch.jit.script
def int_floor(x):
    # type: (float) -> int
    return int(x - x % 1)

print(double_floor(1.2))
print(int_floor(1.2))
print(double_floor(float(5e29)))
print(int_floor(float(5e29)))  # incorrect in torchscript
```